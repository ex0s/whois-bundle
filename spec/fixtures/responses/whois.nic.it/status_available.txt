Domain:             u34jedzcq.it
Status:             AVAILABLE
