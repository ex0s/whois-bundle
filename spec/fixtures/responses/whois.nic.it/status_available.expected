#disclaimer
  %s == nil


#domain
  %s == "u34jedzcq.it"

#domain_id
  %s %ERROR{AttributeNotSupported}


#status
  %s == :available

#available?
  %s == true

#registered?
  %s == false


#created_on
  %s == nil

#updated_on
  %s == nil

#expires_on
  %s == nil


#registrar
  %s == nil

#registrant_contacts
  %s %CLASS{array}
  %s == []

#admin_contacts
  %s %CLASS{array}
  %s == []

#technical_contacts
  %s %CLASS{array}
  %s == []


#nameservers
  %s %CLASS{array}
  %s == []


#response_unavailable?
  %s == false
